runner.go

```diff
@@ -95,6 +101,34 @@ func (r *runner) startRun(c cliContext) error {
 		return nil
 	}
 
+	if c.Bool("se") {
+		os.Setenv("RAINFOREST_RUN_ID", strconv.Itoa(runStatus.ID))
```

Comment:
This won't do what you want, because the environment variable is set in the current process and child processes, but not in the parent.

I don't think it's possible to set environment variables in parent processes without using `ptrace()` or other shinangins. 

I tested this locally with the following program:
```
package main

import "os"

func main() {
        os.Setenv("RAINFOREST_RUN_ID", "1234")
}
```
and doing:
```
go run test.go; echo '$RAINFOREST_RUN_ID'
```
didn't print anything.